xml/product-entities.ent

```diff
@@ -3,8 +3,13 @@
 
 <!-- PRODUCT NAME AND VERSIONS -->
 <!ENTITY productname   "&slehpc;">
-<!ENTITY productnumber "15 SP3">
 <!ENTITY this-version "15 SP3">
+<!ENTITY product-ga    "15">
+<!ENTITY product-sp    "3">
+<!ENTITY productnumbershort "&product-ga;.&product-sp;">
```

Comment:
Are we officially doing `major.minor` for SLE products now? I have never seen this numbering scheme anywhere official (except for Ivo's recent slides but I thought that was a lapse).

Reply:
Ah, this was for something that I've since removed. I originally had a step with a command for installing Package Hub instead of leaving it as a prerequisite, and the command includes the version number `15.3`. 

Now that I'm not using that command, I agree this entity probably isn't required.